### DISCUSSION TITLE
Unable to connect SupaBase with N8N on Docker

### DISCUSSION THREAD

**Discussion by iphillz** [supabase/supabase#19419] [Questions]
I have installed Supabase and N8N on the docker (self hosted) 

I can access Supabase via localhosd. However, when I try to connect N8N to Supabase, I receive an error that the service refused the connection. This suggests that Supabase may be offline.

![image](https://github.com/supabase/supabase/assets/152276393/1bb151d1-9131-4385-8e7e-5237544dd822)

I have followed the documentation while installing Supabase, including generating a new token and new Secret Token I can also access localhost via the 8000 port in my browser. However, I am unable to connect Supabase and N8N.

**Comment by onitblackvolutionai** [comment]
yes same error, did you find the solution to solve it , please help me ?
